### DISCUSSION TITLE
cpptools running button disappears.

### DISCUSSION THREAD

**Discussion by ycnx** [microsoft/vscode-cpptools#10073] [Q&A]
![image](https://user-images.githubusercontent.com/79720579/198314188-a4a5797f-ad1f-4438-ae0f-55abaf96144a.png)
The triangle in the figure above disappears.
I don't know how to solve it, reinstalling expansion is invalid.

Now, it's like this, but I only according to press F5 to run cpp:
![image](https://user-images.githubusercontent.com/79720579/198315336-1eae7b0b-9f2a-46db-8fac-79024333024f.png)

**Comment by sean-mcmanus** [comment]
What file do you have active when the button disappears? What triggers it to disappear? Are you sure the active file is a C or C++ file? Is the C_Cpp.debugShortcut setting set to true? Is the document of schema type "file"?

**Reply by ycnx** [reply]
It appeared! I don't know when and why the setting of the debugshortcut is automatically canceled, but the problem is solved, thank you!
